dev/com.ibm.ws.cdi.mp.context/resources/com/ibm/ws/cdi/mp/context/resources/CDI_MP_CONTEXT.nlsprops

```diff
@@ -26,8 +26,8 @@
 # com.ibm.ws.concurrent.mp.1.0 message bundle
 
 # do not translate: CDI, CompletionStage
-CWWKC1158.cannot.lazy.enlist.beans=CWWKC1158E:You must access CDI beans prior to capturing context for the contextual action or task in order for the CDI \
-beans to behave correctly when accessed from the action or task. The following CDI beans must be accessed before capturing context: {0}.
+CWWKC1158.cannot.lazy.enlist.beans=CWWKC1158E:You must access the CDI beans before context is captured for the contextual action or task so that the CDI \
+beans to behave correctly when accessed from the action or task. The following CDI beans were not accessed before the context was captured: {0}.
```

Comment:
A grammar error is introduced by this change:
"so that the CDI beans to behave correctly"
The word "to" needs to be removed.